protos/payload.proto

```diff
@@ -18,10 +18,22 @@ syntax = "proto3";
 import "rule.proto";
 
 message TransactionPayload {
-    CreateAccount create_account = 1;
-    CreateAsset create_asset = 2;
-    CreateHolding create_holding = 3;
-    CreateOffer create_offer = 4;
+
+    enum PayloadType {
+        CREATE_ACCOUNT = 0;
```

Comment:
Use
```
TYPE_UNSET = 0;
```
And then start `CREATE_ACCOUNT` at `1`.

Reply:
What is the need for TYPE_UNSET?